### ISSUE TITLE
Questions about training

### ISSUE THREAD

**Issue by christk1** [luckycallor/InsightFace-tensorflow#23]
Hello @luckycallor and thanks for your implementation, i am trying to apply arc loss in my own dataset but loss drops from around 57 to around 20 and then it stops converging. Furthermore the training accuracy stays at 0.0. I am using a batch size of 448, any tips for this?
And the other question is: for validation, if i measure the accuracy by the traditional way with 
tf.reduce_mean(tf.cast(tf.equal(pred, self.train_labels), tf.float32)), without comparing 2 embeddings at a time, it would be a false prediction? if yes, can you explain me the reason?

**Comment by jiang7030773** [comment]
hi, @christk1, Have you solved the first problem?

**Comment by christk1** [comment]
> hi, @christk1, Have you solved the first problem?

hello, sorry for the late reply, after searching the official implementation on mxnet i found that you must first train with softmax and then finetune with arcloss
